week-3-nestjs-back-end/w03e01-modular-app/food-delivery/src/order/orders/orders.controller.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Controller,
+  Get,
+  HttpException,
+  HttpStatus,
+  Post,
+  Request,
+  UseGuards,
+} from '@nestjs/common';
+import { JwtAuthGuard } from 'src/auth/jwt-auth.guard';
+import { OrdersService } from './orders.service';
+
+@Controller()
+export class OrdersController {
+  constructor(private ordersService: OrdersService) {}
+
+  @UseGuards(JwtAuthGuard)
+  @Get('orders')
+  getOrders(@Request() req) {
+    return this.ordersService.getAll(req.user.id);
+  }
+
+  @UseGuards(JwtAuthGuard)
+  @Post('orders')
+  createOrder(@Request() req) {
```

Comment:
Fajnie by było jakbyś tutaj użył `DTO` do walidacji:
https://docs.nestjs.com/techniques/validation#validation

Dużo by to uprościło i odpakowywania danych i samej walidacji.